.github/workflows/nuget-publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup dotnet
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '5.x'
+          dotnet-version: "5.x"
```

Comment:
To ma jakieś znaczenie?

Reply:
Nie, nawet tego nie musi być, stringi w yml są domyślnie